doc/src/SUMMARY.md

```diff
@@ -16,3 +16,7 @@
 # Contributing
 
 - [Documentation](contributing/documentation.md)
+
+# Message Access Requirements
+
+- [Message Access](message_access.md)
```

Comment:
```suggestion
- [Amplifier Access Control](message_access.md)
```
I think we can merge this into the `Contracts` section, but make it a separate paragraph

Reply:
Added as a subsection to `Contracts`